### PR TITLE
Add /robots.txt

### PR DESCRIPTION
**Pull request by pmac**
Add setting to turn off disallow. Mainly so that we can
tell robots not to index the staging instance.

### CODE REVIEW

File: standup/status/views.py [hunk 1]
```diff
@@ -261,3 +261,12 @@ def csp_violation_capture(request):
...
     return HttpResponse('Captured CSP violation, thanks for reporting.')
+
+
+def robots_txt(request):
+    if settings.ROBOTS_ALLOW:
+        content = ''
+    else:
+        content = 'User-agent: *\nDisallow: /'
+
+    return HttpResponse(content, content_type='text/plain')
```
Comment (willkg) [inline standup/status/views.py]: When we get caching going, we should totally cache this.

**Comment by willkg** [comment]
This looks good to me. r+

File: standup/status/views.py (hunk 1 above)
Comment (pmac) [inline standup/status/views.py]: I could add some cache headers, so at least some clients would cache it.

Comment (willkg) [inline standup/status/views.py]: We could do that... but I think we could also just create an issue for "let's cache things that make sense to cache" and push it off until then.

Either way is fine with me.

**Pull request merged**